how_to_build.md

```diff
@@ -41,7 +43,7 @@ Using a command prompt, navigate to your clone root folder and execute:
 
 `bundle exec rake`
 
-This executes the default build tasks. After the build has completed, the build artifacts will be located in `Build`.

```

Comment:
I just checked TC and it did indeed have some redundant Build/ entries in its artifacts paths. I trimmed it down to just `artifacts\**\*.*`.